.github/workflows/windows-build.yml

```diff
@@ -25,26 +25,24 @@ jobs:
           channel-priority: strict
           activate-environment: pyos-build
           environment-file: .github/environment.yml
-      - name: Install libpgmath
-        shell: bash -l {0}
-        run: |
-          conda activate pyos-build
-          conda install libpgmath
       - name: Build and install pyoptsparse
         shell: cmd /C CALL {0}
         run: |
           call conda activate pyos-build
-          set IPOPT_DIR=%CONDA_PREFIX%\Library
 
-          # We want to use the clang linker `ld-lld.exe`
-          # The conda-forge clang packages set the LD environment variable and LDFLAGS for that linker.
-          # But meson doesn't respect the LD environment variable - so set `<compiler entry>_LD`
-          # https://mesonbuild.com/howtox.html#set-linker
-          set CC_LD=%LD%
-          set CXX_LD=%LD%
-          set FC_LD=%LD%
+          :: debug - list conda environment
```

Comment:
This debug info is very useful. I see no cost to leaving these lines in, as it only affects the windows build logs on CI.